src/stdlib_stats_distribution_uniform.fypp

```diff
@@ -0,0 +1,482 @@
+#:include "common.fypp"
+#:set RC_KINDS_TYPES = REAL_KINDS_TYPES + CMPLX_KINDS_TYPES
+#:set ALL_KINDS_TYPES = INT_KINDS_TYPES + RC_KINDS_TYPES
+Module stdlib_stats_distribution_uniform
+    use stdlib_kinds
+    use stdlib_error, only : error_stop
+    use stdlib_stats_distribution_PRNG, only : dist_rand
+
+    implicit none
+    private
+
+    real(dp), parameter  :: MESENNE_NUMBER = 1.0_dp / (2.0_dp ** 53 - 1.0_dp)
```

Comment:
should it be defined for `qp`?

Reply:
No. The method is only for 64-bit, can't extrapolate to 128-bit.